activities/Abecedarium.activity/lib/l10n.js

```diff
@@ -0,0 +1,58 @@
+define(['i18next.min', 'axios.min'], function (i18next, axios) {
+    const l10n = {};
+    let initialized = false;
+
+    l10n.init = async (lang) => {
+        await i18next.init({
+            lng: lang,
+            debug: true,
```

Comment:
Will be better to disable i18next debug information.  Was not there in the initial file https://github.com/llaske/sugarizer/blob/dev/activities/QRCode.activity/lib/l10n.js#L6